### DISCUSSION TITLE
Version suggestion form

### DISCUSSION THREAD

**Discussion by kous500** [kous500/CurveBuilding#31] [Suggestion Form]
Please comment your desired version here.
Example: forge 1.11.2

**Comment by kous500** [comment]
**Currently most requested versions:**
- Fabric 1.19.2 ~ 1.19.3
- Bukkit and Fabric 1.12.2

**Version requested at least once:**
- Fabric 1.20.6
- Fabric 1.20.4
- Fabric 1.18.2
- Fabric 1.14.4
- Any Forge version
- BE version!?

**Reply by JustN00ne** [reply]
It would be really nice it the mod would get the 1.19.2 cause there would be a lot more downloads! My own server is on 1.19.2 and its a creative server (so that would be a massive addition). I have experimene porting 1.18.2 mods to 1.19.2 but not 1.19.4+ mods. And i would beg you to really do that, i would be really thankful

**Reply by JustN00ne** [reply]
Hello @kous500 , are there current updates ongoing, or are you taking a break. Lmk so i can notify my server, we are waiting for the 1.19.2 version since a long time lol. Wish you luck

**Comment by StarboardPluger** [comment]
Hello! Is it possible to make it fabric 1.18.2 compatible? Thank you!

**Comment by piggybxnk** [comment]
Fabric 1.20.4?

**Comment by Foolish69420** [comment]
Fabric 1.14.4 Im begging.

**Comment by quadmoo** [comment]
Fabric 1.21.4 please!

**Comment by Rolling-Catawa** [comment]
Fabric 1.19.2 pls

**Comment by Wargroy** [comment]
Paper/Purpur 1.21.7+

**Comment by matteonapoleao-debug** [comment]
Forge 1.20.1 please

**Comment by flouchgamer88-hub** [comment]
Me and my friend are working a project that in the near future we will be making lots of roads. But we want this to be as quick and efficient and as good looking as we can, sadly when I found this mod/plugin there is no version for 1.20.1 forge or any forge version at all. 

If you could make a version for Forge 1.20.1 that would be very nice.

**Comment by Rafael225** [comment]
PLEASE, do for 1.18.2 forge, it's a very recent release and have many mods to this release. I have tested the engine in 1.20.1, is very nice, but the most mods is for 1.18.2, like as it once was 1.16.5, and 1.12, and 1.7.10... Please, I need this mod!!

**Comment by KitsuNek0** [comment]
Fabric 1.20.4 if possible

**Comment by Neon4UltraPlays** [comment]
**Request: Forge 1.20.1**
Hello CurveBuildingMod developers, I am a player using the mod for a passion project of mine that involves placing and using various curves to create major expressway plans. However, after losing my world to the corruption of loading it into a new version, having to recreate everything, this mod saved me from the tedious work of building curves. Sadly, this mod only works for Fabric, so I need to switch between Forge and Fabric manually. Additionally, the majority of my mods are Forge, so after placing all the Forge mod blocks, it won't load in Fabric.

If a Forge version of 1.20.1 exists, it would solve all my problems of switching between two modding versions and worrying about having to rebuild/construct everything in Fabric before using it all on Forge. If it gets added, I am able to use the mod alongside my Forge versions.

Thank you!
<img width="2559" height="1332" alt="Screenshot 2025-06-22 135726" src="https://github.com/user-attachments/assets/62bea3d4-82b0-4a71-83a4-4e05339349b9" /> <img width="1601" height="870" alt="image" src="https://github.com/user-attachments/assets/e7aebc3a-9be2-46ab-af3e-de2c07d78f72" /><img width="1586" height="1104" alt="Screenshot 2025-07-09 200946" src="https://github.com/user-attachments/assets/79d65a65-046e-455b-b5e8-5c053e6b6a94" />

**Comment by ri3333333** [comment]
Hello! Is it possible to make it fabric 1.21.11 compatible?

**Comment by Wargroy** [comment]
1.21.11 bug fixes and 26.1 support minecraft java purpur/paper

I gave you a donation, thank you!

**Comment by Wargroy** [comment]
Could you please add up to 26.1.2 support? @kous500

**Reply by Wargroy** [reply]
I've donated. Please update to newest minecraft version 26.2. Thanks! @kous500 
Please also fix the SendParticle for other players to see the line connecting the dots.

**Comment by Wargroy** [comment]
@kous500 Could you please confirm when plugin will be updated to Paper 26.2+?

**Comment by Wargroy** [comment]
@kous500 Could you please update to 26.2+? 26.3?
